.jshintrc

```diff
@@ -1,10 +1,19 @@
 {
   "node": true,
+	"browser": true,
+	"esversion": 6,
 
   "curly": true,
   "latedef": true,
-  "quotmark": true,
```

Comment:
It appears to me that  you guys have replaced `jshint` with `eslint`.  My editor of choice of `brackets` which uses to `jshint`.  I made above changes to limit the number warnings produced by the editor.

I thought it was better to change the config, which you do not use, than making lots of the changes to the code that has nothing to do with the bug I was trying to fix.